### PR TITLE
Move NEP10 to manifest

### PR DESCRIPTION
**Pull request by shargon**
Close https://github.com/neo-project/neo/issues/779
Wait for amend https://github.com/neo-project/proposals/blob/master/nep-10.mediawiki

### CODE REVIEW

**Comment by erikzhang** [comment]
Move it to manifest?

File: src/neo/Ledger/ContractState.cs [hunk 1]
```diff
@@ -82,7 +82,7 @@ public JObject ToJson()
 
         public StackItem ToStackItem(ReferenceCounter referenceCounter)
         {
-            return new Array(referenceCounter, new StackItem[] { Script, HasStorage, Payable });
+            return new Array(referenceCounter, new StackItem[] { Script, Manifest.ToString(), HasStorage, Payable });
```
Comment (shargon) [inline src/neo/Ledger/ContractState.cs:85]: We need to ensure that the max of this item, it's allowed in the VM

**Comment by erikzhang** [comment]
We should modify https://github.com/neo-project/proposals/pull/114 first.

**Comment by shargon** [comment]
And NEP10 ?

**Comment by erikzhang** [comment]
> And NEP10 ?

Done.

**Pull request merged**